src/acorn_version.h

```diff
@@ -0,0 +1,4 @@
+#ifndef SRC_ACORN_VERSION_H_
```

Comment:
Can we add a comment here and in undici_version.h mentioning that these files are generated by `update-acorn.sh`/`update-undici.sh` so that people don't attempt to update them manually?

Reply:
Adding

Reply:
Have added the comments @joyeecheung